tokio/src/runtime/tests/mod.rs

```diff
@@ -1,40 +1,7 @@
 //! Testing utilities
 
-#[cfg(not(loom))]
-pub(crate) mod backoff;
```

Comment:
Some tests are moved into other modules (like `tokio::task::tests`) and shared test utilities are moved to `tokio::tests`.